ql/src/java/org/apache/hadoop/hive/ql/security/authorization/command/CommandAuthorizerV2.java

```diff
@@ -121,6 +142,24 @@ private static void addPermanentFunctionEntities(SessionState ss, List<ReadEntit
     return hivePrivobjs;
   }
 
+  private static boolean isDeferredAuthView(Table t){
```

Comment:
Can you add a java doc of this method. May be I am missing the context here. What does deferred authorization mean?